scss/footer.scss

```diff
@@ -0,0 +1,3 @@
+footer {
```

Comment:
Let's rename this file to `_Footer.scss`. See [here](https://trello.com/c/rc0ovqq8/21-dev-refactor-file-names-and-add-eslint-rule) for the new naming convention